guides/v2.1/cloud/configure/setup-cron-jobs.md

```diff
@@ -38,7 +38,7 @@ The following example is the default cron included for {{site.data.var.ece}}.
 crons:
     cronrun:
         spec: "* * * * *"
-        cmd: "php bin/magento cron:run"
+        cmd: "bin/magento cron:run"
```

Comment:
These are YAML file snippets and need the PHP

Reply:
> These are YAML file snippets and need the PHP

**reverted**